tests/ReceivesEventTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace DigitalRisks\LaravelEventStore\Tests;
+
+use DigitalRisks\LaravelEventStore\Contracts\CouldBeReceived;
+use DigitalRisks\LaravelEventStore\Contracts\ShouldBeStored;
+use DigitalRisks\LaravelEventStore\Tests\Traits\InteractsWithEventStore;
+use DigitalRisks\LaravelEventStore\Traits\AddsLaravelMetadata;
+use DigitalRisks\LaravelEventStore\Traits\ReceivedFromEventStore;
+use DigitalRisks\LaravelEventStore\Traits\SendsToEventStore;
+
+class ReceivesEventTest extends TestCase
+{
+    use InteractsWithEventStore;
+
+    /** @test */
+    public function it_sets_the_event_record_on_received_event_when_event_connection_is_sync()
+    {
+        // Arrange.
+        config(['eventstore.connection' => 'sync']);
+
+        $event = new class implements CouldBeReceived {
```

Comment:
wtf

Reply:
Lmao! I quite like this

Reply:
anonymous classes are great for tests! Can see what you're testing right there in the test rather than bottom of file / separate files....

Reply:
Declaration separate from initialization. I'm ok with this, until I'll find a way to create events with factories ;)